### PR TITLE
Integrate acceptance test automation

### PR DESCRIPTION
**Pull request by jon-ruckwood**
Integrate automation from Max's `acc-test-actions` branch and allow acceptance tests to run on GitHub Actions.

### CODE REVIEW

**Review by night0wl — APPROVED** [review]
LGTM 👍

**Comment by jon-ruckwood** [comment]
I'll merge this once #6 is merged.

**Pull request merged**
